src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/getUserDashboardState.tsx

```diff
@@ -331,6 +331,24 @@ export const getUserDashboardState = (
       isPremiumUser: true,
       scanInProgress: false,
     }) ||
+    /**
+     * - US user
+     * - Premium
+     * - Unresolved breaches
+     * - Scan: Unresolved
+     */
+    isMatchingContent(contentProps, {
+      isRelevantGuidedStep:
+        relevantGuidedStep.id === "Scan" ||
+        relevantGuidedStep.id === "DataBrokerManualRemoval",
+      hasExposures: true,
+      hasUnresolvedBreaches: true,
+      hasUnresolvedBrokers: true,
+      isEligibleForFreeScan: false,
+      isEligibleForPremium: false,
+      isPremiumUser: true,
+      scanInProgress: false,
+    }) ||
```

Comment:
This condition matches the one on `L229-238`, but without `relevantGuidedStep.id === "DataBrokerManualRemoval"`. Either we remove the condition above (which looks like the right thing to do) or drop the `relevantGuidedStep.id === "Scan"` on `L341`.